tests/flow/comments/babel-only/jsfmt.spec.js

```diff
@@ -0,0 +1 @@
+run_spec(__dirname, ["babel-flow"]);
```

Comment:
This can't test on `flow` parser, https://github.com/prettier/prettier/issues/9564

Reply:
Changed the tests to avoid this bug.